src/components/Login/Login.css

```diff
@@ -0,0 +1,108 @@
+.login-wrapper {
```

Comment:
The background files should be in a folder name 'assets' and you should prefer png files because they are lighter